lib/Starman/Server.pm

```diff
@@ -77,7 +77,7 @@ sub run {
             my($h, $p, $opt) = split /:/, $listen, 3;
             $listen{host} = $h if $h;
             $listen{port} = $p;
-            $listen{proto} = 'ssl' if 'ssl' eq lc $opt;
+            $listen{proto} = 'ssl' if 'ssl' eq (defined $opt ? lc $opt : "");
```

Comment:
simpler to write `if defined $opt && $opt eq 'ssl'`

Reply:
Well, `if defined $opt and 'ssl' eq lc $opt` to be exact. (You lost the `lc`.)